### ISSUE TITLE
Tor hidden services for RPC

### ISSUE THREAD

**Issue by Kixunil** [Kixunil/cryptoanarchy-deb-repo-builder#24]
This is probably the most robust (user-friendly) way to connect to a remote node. There should be packages that allow seamless integration between services, Tor and desktop.

**Comment by Fonta1n3** [comment]
Agree! I have been focusing mostly on ios/macos([FullyNoded](https://github.com/Fonta1n3/FullyNoded/blob/master/Readme.md) and [StandUp](https://github.com/BlockchainCommons/Bitcoin-Standup)) but started working on a linux script lately that sets a user up with bitcoind and tor with a V3 HS controlling the rpcports, with the end goal of displaying a QR code to the user for remote connection. Please check out the standard we are following which a number of other projects have supported: https://github.com/BlockchainCommons/Bitcoin-Standup#quick-connect-url-using-btcstandup cc @ChristopherAllen

Great to see other like minded projects! Look forward to diving more into what you've done here, I can probably learn a lot :)

**Comment by Kixunil** [comment]
Awesome, URI scheme is something I was thinking of too! I will need to think about it more deeply to provide any feedback, since I didn't analyze it properly yet. At the first glance label is a great idea.

I will want to somehow make sure that user only has to connect to a remote machine once, not scanning multiple times for each service (Core, Electrum, LN), but maybe you handle that already?
